x-pack/test/functional/apps/code/explore_repository.ts

```diff
@@ -85,6 +86,15 @@ export default function exploreRepositoryFunctonalTests({
         });
 
         await testSubjects.click('codeFileTreeNode-Directory-src');
+
+        await retry.tryForTime(1000, async () => {
```

Comment:
I would suggest increasing the timeout more (maybe 5000?) in case any slow responses.

btw, the current `feature/merge-code` sometimes has timeouted test on `workspace_handler`: https://kibana-ci.elastic.co/job/elastic+kibana+pull-request/7132/JOB=x-pack-intake,node=immutable/console

can you help to increase the timeout there too?

Reply:
I doubt it can be fixed simply by increasing the timeout. It's a jest+nodegit bug we found before, moving that test to mocha.